src/includes/blog-article.njk

```diff
@@ -0,0 +1,24 @@
+<li class="blog-article">
+	<div class="blog-article-featured-image">
+		<img src="{{ postFeaturedImage }}">
+	</div>
+
+	<h2 class="blog-article-title">
+		<a href="{{ postPermalink }}">{{ postTitle }}</a>
+	</h2>
+
+	<p class="blog-article-date">
+		<time datetime="{{ postDate }}">{{ postDate }}</time>
+	</p>
+
+	<div class="blog-article-excerpt">
+		{{ postExcerpt | safe }}
+
+		<p class="blog-article-read-more">
+			<a class="blog-article-read-more-link" href="{{ postPermalink }}">
+				Read more
+			</a>
+			&nbsp;<span class="blog-article-read-more-arrow">▸</span>
```

Comment:
Could this be done with content via CSS instead since it's primarily presentational?

Reply:
I'll look into that